tests/Makefile.tests

```diff
@@ -13,30 +13,25 @@ CFLAGS+=-I$(SOLO5_DIR)
 %.o: %.c $(HEADERS)
 	$(CC) $(CFLAGS) -c $< -o $@
 
-ifdef UKVM_TARGETS
 Makefile.ukvm: $(UKVM_SRC)/ukvm-configure
 	$(UKVM_SRC)/ukvm-configure $(UKVM_SRC) $(UKVM_MODULES)
 
-include Makefile.ukvm
+.PHONY: ukvm-clean
+-include Makefile.ukvm
```

Comment:
@hannesm is the `-include` directive only respected by certain versions of Make?  On my machine it seems to always build `Makefile.ukvm` so that it can include it?